recipes/usearch/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+# To test, please run: conan create all/conanfile.py --version 0.8.0
+import os
+
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import get, copy
+from conan.tools.layout import basic_layout
+
+
+required_conan_version = ">=1.53.0"
+
+
+class USearchConan(ConanFile):
+    name = "usearch"
+    license = "Apache-2.0"
+    description = "Smaller & Faster Single-File Vector Search Engine from Unum"
+    homepage = "https://github.com/unum-cloud/usearch"
+    topics = ("search", "vector", "simd")
+    settings = "os", "arch", "compiler", "build_type"
+    url = "https://github.com/conan-io/conan-center-index"
+    package_type = "header-library"
+
+    # No settings/options are necessary, this is header only
+    # Potentially add unit-tests in the future:
+    # https://docs.conan.io/1/howtos/header_only.html#with-unit-tests
+    # no_copy_source = True
```

Comment:
```suggestion
    no_copy_source = True
```
Please, remove this comment as it points to Conan 1.x documentation. It's outdated.
Plus, keep `no_copy_source` as we do not need to re-copy files from source to build folder when packaging header-only, we just copy from source folder to package folder.